lib/v1.mapper.js

```diff
@@ -0,0 +1,343 @@
+/**
```

Comment:
we can -- just curious to hear why either one matters? I've seen in other integration code where we have this kind of naming convention when having multiple versions.